### ISSUE TITLE
upside down

### ISSUE THREAD

**Issue by jemzym** [OpenShot/openshot-qt#1590]
i added my video to the software to start editing it and its upside down, every editor i have used so far has had other problems but not with it being upside down 
![2018-05-14](https://user-images.githubusercontent.com/39276912/40014958-dce9e0cc-57a9-11e8-8695-0a0136901f14.png)

**Comment by jemzym** [comment]
No, i used movie maker and i tried to change the to mp4 and it corrupted the file. So i triee this editing software and the only thing wrong was it being upside down.

**Comment by peanutbutterandcrackers** [comment]
I would like to suggest that you [use the latest daily build](https://gist.github.com/peanutbutterandcrackers/3618493b99ba08003e4e4b1602459914). A lot of the issues have been resolved in the daily builds and OpenShot is a lot more stable. Maybe using the latest daily build will fix it.

If that doesn't quite work, you might want to transcode your video using [HandBrake](https://handbrake.fr/).

Hope that helps!

**Comment by peanutbutterandcrackers** [comment]
You could also Right click on the clip once you drag it into the timeline and click on Rotate > Rotate 180 (flip). That should fix it, too.
![screenshot at 2018-05-15 17-38-00](https://user-images.githubusercontent.com/26764547/40055170-c503129a-5866-11e8-9007-ee1bae704fb8.png)

**Comment by jemzym** [comment]
Thank you, ill try it
